### ISSUE TITLE
Dotkernel Frontend installation giving problems

### ISSUE THREAD

**Issue by goharsahi** [dotkernel/frontend#218]
While installing dotkernet/frontend on my windows 10 PC, it gives following problems.

```
Installing dotkernel/frontend (3.1.0)
  - Downloading dotkernel/frontend (3.1.0)
  - Installing dotkernel/frontend (3.1.0): Extracting archive
Created project in D:\datad\ziwajun-muneerah\ziwajun_muneerah
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - laminas/laminas-dependency-plugin[1.0.0, ..., 1.0.4] require composer-plugin-api ^1.1 -> found composer-plugin-api[2.0.0] but it does not match the constraint.
    - Root composer.json requires laminas/laminas-dependency-plugin ^1.0 -> satisfiable by laminas/laminas-dependency-plugin[1.0.0, ..., 1.0.4].
You are using Composer 2, which some of your plugins seem to be incompatible with. Make sure you update your plugins or report a plugin-issue to ask them to support Composer 2.
```

**Comment by goharsahi** [comment]
The composer version:

```
D:\dataD\ziwajun-muneerah\ziwajun_muneerah>composer self-update --stable
Upgrading to version 2.0.11 (stable channel).

```

**Comment by Calin23** [comment]
Hi @goharsahi please confirm that you are using  "laminas/laminas-dependency-plugin": "^2.1", in your composer.json.

**Comment by Calin23** [comment]
@goharsahi apparently if you install the project with _"composer create-project dotkernel / frontend -s dev dk"_ command and use any of _[ stable, RC, beta, alpha ]_, instead of "dev" the installation will fail.

**Comment by goharsahi** [comment]
@Calin23 The issue was with ```laminas/laminas-dependency-plugin``` ```^1.0```... I changed it to ```^2.0``` as @thomas suggested me [here](https://laminas.slack.com/archives/C4RF57B0E/p1616245094015500) and this fixed the issue. 

This should be fixed in the repo also.

**Comment by goharsahi** [comment]
I see its already fixed in the repo, but I wonder where's it downloaded from when I ran through composer the first time. I also noticed it was using older version of twig.

```
 - Removing twig/extensions (v1.5.4)
  - Upgrading twig/twig (v2.14.4 => v3.3.0)
```

**Comment by arhimede** [comment]
Yes, is something that need to be checked and fixed.

In the documentation, is stated that the proper command should be :

`$ composer create-project dotkernel/frontend -s dev dk`

But on packagist page, the *default* command is 
`composer create-project dotkernel/frontend`
which crash

**Comment by arhimede** [comment]
Fixed in release 3.2.0

It seem that we didn't made a release in long time.
